contrib/NeonVM/cell.c

```diff
@@ -125,12 +125,48 @@ Cell *cell_createArrayCell(uint32_t iElements)
     if (c->array == NULL) {
         fatal_error("Unable to allocate memory for array.");
     }
-    for (uint32_t i = 0; i < c->array_size; i++) {
+    for (uint64_t i = 0; i < c->array_size; i++) {

```

Comment:
Better decide how big you want your arrays to be here. Up in cell_fromArray, you use a uint32_t.